src/AspNet.Security.OAuth.QuickBooks/QuickBooksAuthenticationOptions.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using static AspNet.Security.OAuth.QuickBooks.QuickBooksAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.QuickBooks
+{
+    /// <summary>
+    /// Defines a set of options used by <see cref="QuickBooksAuthenticationHandler"/>.
+    /// </summary>
+    public class QuickBooksAuthenticationOptions : OAuthOptions
+    {
+        private readonly string scopeValOpenId = "openid email profile phone address";
```

Comment:
It would seem to me that the _minimum_ required scopes are `openid` and `profile`. The rest are extras.

Reply:
The email and phone scopes are necessary.